vlib/v/debug/tests/interface_var.vv

```diff
@@ -17,6 +17,19 @@ fn interface_var(a ITest) {
 	}
 }
 
+struct MyTest {
+	t ITest
+}
+
 fn main() {
 	interface_var(Test{ a: true })
+	t1 := Test{}
+	t2 := MyTest{
+		t: t1
+	}
+	mut t3 := Test{}
+	t4 := MyTest{
+		t: t3
+	}
+	$dbg;
```

Comment:
This part is not tested in `vlib/v/debug/tests/interface_var.expect` ; the expect script ends after the first `$dbg;` statement in `interface_var()`, and the later `$dbg;` will be compiled, but will not run.